t/04-scheduler.t

```diff
@@ -14,8 +14,6 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
-use strict;
-use warnings;
```

Comment:
If you want to use a module to activate strictures it has to be at the beginning of the file. Because strictures are only active from the point they were activated, not before.

Reply:
good, thx.

Reply:
done